wrappers/wrapper.rb

```diff
@@ -56,7 +56,7 @@ def assert_vehicles_start_or_end(vrp)
 
     def assert_vehicles_no_capacity_initial(vrp)
       vrp.vehicles.none?{ |vehicle|
-        vehicle.capacities.find{ |c| c.initial&.positive? }
+        vehicle.capacities.any?{ |c| c.initial && c.initial < c.limit }
```

Comment:
I don't quite get this condition.

Reply:
Does `initial_load` signify the existing load in the vehicle? If so, how come it can be greater than the vehicle capacity?

Reply:
If the initial capacity is greater than the capacity, it supposes or that we have a overload multiplier, or the initial capacity is not restricted

Reply:
My question was due to the definition miss-match :) initial-load vs initial-capacity. it makes sense now